babel.config.js

```diff
@@ -0,0 +1,100 @@
+// eslint-disable-next-line func-names
+module.exports = function (api) {
+  const validEnv = ['development', 'test', 'production'];
+  const currentEnv = api.env();
+  const isDevelopmentEnv = api.env('development');
+  const isProductionEnv = api.env('production');
+  const isTestEnv = api.env('test');
+
+  if (!validEnv.includes(currentEnv)) {
+    throw new Error(
```

Comment:
- [ ] Kindly in the babel.config.js, consider adding the following as in the guide
```javascript
module.exports = {
  presets: [
    [
      '@babel/preset-env',
      {
        targets: {
          node: 'current',
        },
      },
    ],
    '@babel/preset-react',
  ],
};
```

Reply:
Hello, thank you for suggestions, however, **I've already added that part into my code** as you can see on the screenshot below, also due to compilation other code also was added automatically 😃 🥂. 

![RoR-React-Required](https://user-images.githubusercontent.com/57558480/198725449-f70983ca-2cd2-454d-8b98-83dbb51742c9.png)